Rubberduck.RibbonDispatcher/AbstractRibbonDispatcher/RibbonToggle.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using Microsoft.Office.Core;
+
+namespace RubberDuck.RibbonDispatcher {
+    using LanguageStrings     = IRibbonTextLanguageControl;
+
+    using ClickedEventHandler = EventHandler<ClickedEventArgs>;
```

Comment:
These `using` statements obscure the code that use them IMO. I'd just spell it out wherever needed.